accounts/abi/type.go

```diff
@@ -46,6 +46,7 @@ type Type struct {
 	Kind reflect.Kind
 	Type reflect.Type
 	Size int
+	DecimalSize int // Determines the length of the binary coded decimal in fixed point types. 
```

Comment:
Please don't add this here just yet if it's not used by any code. Hold it off when associated code gets added.